desktop/cmp/form/FormField.js

```diff
@@ -80,7 +80,17 @@ export class FormField extends Component {
          * Focus or toggle input when label is clicked.
          * Defaulted from containing Form, or true.
          */
-        clickableLabel: PT.bool
+        clickableLabel: PT.bool,
+
+        /**
+         * Alignment of label text, default 'left'.
+         */
+        labelAlign: PT.oneOf(['left', 'right']),
```

Comment:
This seems like it could easily be confused with the overall position of the label relative to the input.  I would propose that this be customizable via CSS instead of a prop.  

In general, I have been trying to keep in mind that CSS exists as a very configurable, easily controlled system for style overrides - provided we easy-to-select classes in place consistently for our components.  Would apply to labelWidth as well - I almost added that last week, then realized that a CSS rule was easy.  

Having the inline styles could make adjustments trickier to target, given that they will get a high level of specificity out of the box.  Thoughts on unwinding?

Reply:
Also thinking that we can use CSS targeted at the rest-form to make the labels stand out a bit better in this context, without over-committing to formField changes.  E.g.
```
.xh-rest-form .xh-form-field-label {
  text-align: right;
  font-weight: bold;
  width: 130px;
  margin-right: var(--xh-pad-px);
}
```